src/styles/main.scss

```diff
@@ -4,6 +4,7 @@ html {
 
 body {
   margin: 0;
+  transition: color 300ms, transform 300ms;
```

Comment:
hmm why did you delete the transition from link and card? remove transition from body and add them back
but specify the properties you want to animate

Reply:
My bad. Fixed